src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -47,6 +48,12 @@ static const int32_t trigger_matches[] = {
 	SR_TRIGGER_ONE,
 };
 
+static const char* external_clock_edges[] = {
+	"r", // positive edge
+	"f"  // negative edge
```

Comment:
Maybe just say "rising" and "falling" as this is what "r" and "f" stand for?

Reply:
Good point! Will do.